cli/index_data.py

```diff
@@ -80,6 +80,33 @@ def get_document_generator(
             **{"document_description_embedding": embeddings[0, :].tolist()},
         }
 
+
+def get_text_document_generator(
```

Comment:
Pedantic but my understanding is that this function filters tasks based upon whether they match the input vars for content type and translation type and then creates an iterable generator from this. Do you think the comment / name should reflect the filter step?